extensions/machine-learning/src/common/constants.ts

```diff
@@ -44,11 +44,28 @@ export const registeredModelsTableName = 'registeredModelsTableName';
 export const rPathConfigKey = 'rPath';
 export const adsPythonBundleVersion = '0.0.1';
 
+// TSQL
+//
+
+// The data types that are supported to convert model's parameters to SQL data
```

Comment:
is there any public documentation mentioning these types, if so please add a link so that it can be easily referred and updated if needed.

Reply:
we don't have any doc today but will work to create one